.github/workflows/actions.yml

```diff
@@ -70,6 +70,7 @@ jobs:
         run: |
           echo "VERSION=${{ matrix.clickhouse-version }}" > tests/.env
           if [[ "${{ matrix.clickhouse-version }}" > "21.7" ]]; then echo "ORG=clickhouse"; else echo "ORG=yandex" ; fi  >> tests/.env
+          if [[ "21.10" = "`echo -e "21.10\n${{ matrix.clickhouse-version }}" | sort -V | head -n1`" ]]; then echo "TOP_LEVEL=clickhouse"; else echo "TOP_LEVEL=yandex" ; fi  >> tests/.env
```

Comment:
Can we replace `"21.10" = "``echo -e "21.10\n${{ matrix.clickhouse-version }}" | sort -V | head -n1``"` with `"${{ matrix.clickhouse-version }}" >= "21.10"`?

And the same in another place.

Reply:
No, we cannot. Because bash makes alphanumeric comparison, not version-wise. Please check `if [[ "21.9" < "21.12" ]]; then echo "fine"; else echo "fail"; fi`